### DISCUSSION TITLE
Github Pages

### DISCUSSION THREAD

**Discussion by Eunice-K-Ewudzie** [community/community#66536] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need help. I cannot see my page after deploying on github pages. Tried all that I can.

**Comment by radar-blip** [comment]
It's because your index.html is inside a folder. Just take it out of the folder and you should be all set.
GitHub Pages cant find it.

Also, you will need to change this link (in index.html): 
`<img src="/assets/logo.svg" alt="picture of logo">`

to this:
`<img src="assets/logo.svg" alt="picture of logo">`

**Comment by Eunice-K-Ewudzie** [comment]
Oh ok. Thank you. Will do that and see.

**Comment by Eunice-K-Ewudzie** [comment]
It worked. Thanks
Ewudzie Eunice Kakra Ama
AMALITECH Full Stack Trainee
+233559334284
***@***.***
